src/components/CustomerProfile.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const CustomerProfile = ({ profileID, guestID }) => {
+  return profileID !== null ? (
+    <section className="customer-profile">
+      <h3 className={guestID ? "loading" : "loaded"}>
+        Loading Customer Profile
+      </h3>
+      <p>
+        Guest First Name:<span>{profileID.firstName}</span>
+      </p>
+      <p>
+        Guest First Name:<span>{profileID.surname}</span>
+      </p>
+      <p>
+        Guest Status:<span>{profileID.vip ? "VIP" : "Normal"}</span>
+      </p>
+      <p>
+        Guest Phone Number:<span>{profileID.phoneNumber}</span>
+      </p>
+      <p>
+        Guest Email:<span>{profileID.email}</span>
+      </p>
+    </section>
+  ) : (
+    <h3>Loading</h3>
```

Comment:
This `Loading` message shows on the page before the user clicks on a profile, which is confusing.

I think you want to move the display of the `Loading` text a level up to the `SearchResults` component instead. For this component, instead of using the ternary operator, i'd use the logical and to render only if there are profile details.

E.g.
```
return profileID !== null && (
    <section className="customer-profile">
     .......
    </section>
  )
```